overlord/servicestate/service_control_test.go

```diff
@@ -210,6 +211,12 @@ func verifyControlTasks(c *C, tasks []*state.Task, expectedAction, actionModifie
 			sort.Strings(obtainedServices)
 			sort.Strings(bySnap[sa.SnapName])
 			c.Assert(obtainedServices, DeepEquals, bySnap[sa.SnapName])
+			explicitServices := sa.ExplicitServices
+			if len(expectedExplicitServices) == 0 {
+				c.Assert(explicitServices, HasLen, 0)
+			} else {
+				c.Assert(explicitServices, DeepEquals, expectedExplicitServices)
+			}
```

Comment:
Just pass `nil` instead of `[]string{}` to `verifyControlTasks` for expected services and this can be simplified to just
```
c.Assert(sa.ExplicitServices, DeepEquals, expectedExplicitServices)
```

Reply:
Oh :-)